client/src/main/java/io/obswebsocket/community/client/message/response/stream/GetStreamServiceSettingsResponse.java

```diff
@@ -20,6 +20,7 @@ public GetStreamServiceSettingsResponse() {
   @ToString
   public static class Data {
 
+    private String streamServiceType;
     private JsonObject serviceSettings;
```

Comment:
Do we know the value type here? Maybe we should add a `// TODO`?

Reply:
No we don't, i will add a todo